classes/event/course_module_viewed.php

```diff
@@ -0,0 +1,32 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace mod_cpdlogbook\event;
+
```

Comment:
This has a double whitespace line, and also is missing a phpdoc header, normally this would be picked up by the CI checks so something is probably slightly off in the github tasks

Reply:
split into https://github.com/brendanheywood/moodle-mod_cpdlogbook/issues/68